docs/guides/nginx-configuration.md

```diff
@@ -1,7 +1,9 @@
 ### Notes & Warnings
 
 - **This is an unsupported configuration created by the community**
+- If you're using Raspbian Buster (debian 10), then you should install 'php7.3-fpm' instead of v7.0, and change all instances of 'php7.0-fpm' to 'php7.3-fpm'.
 - If you're using php5, change all instances of `php7.0-fpm` to `php5-fpm` and change `/run/php/php7.0-fpm.sock` to `/var/run/php5-fpm.sock`
+- php sqlite package must be installed otherwise Networking and Querying will throw an error that it can't access the database.
```

Comment:
```suggestion
- The `php7.0-sqlite` package must be installed otherwise Networking and Querying will throw an error that it can't access the database.
```